src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
```

Comment:
- Please not that your `App.js` file should also be class based for this milestone 👍🏽